experiments/opencog/pattern_matcher_vqa/pattern_matcher_vqa.py

```diff
@@ -332,9 +332,22 @@ def answerQuestionByImage(self, image, question, use_pm=True) -> QueryProcessing
             otherwise unified rule engine will be used
         :return: QueryProcessingData
         """
+        features, boxes = self.featureExtractor.getFeaturesByImage(image)
+        return self.answerQuestionByFeatures(features, boxes, question, use_pm)
+
+    def answerQuestionByFeatures(self, features, boxes, question, use_pm=True):
```

Comment:
Previously ready-to-use features were loaded from file by specific feature extractor implementation. What is the reason of introducing new method and passing features as arguments?